README.md

```diff
@@ -79,64 +79,29 @@ _(This is where you begin the server setup instructions)_
 	`cd /usr/local/devcycle`
 
 	`sudo bash setup.sh` _(This will take a significant amount of time on mod_wsgi-httpd, 10-20 mins)_
-6. Create a virtual host & WSGI file for the Apache server to display the Django application by creating a new .conf file.
+6. Create a virtual host & WSGI file for the Apache server to display the Django application by creating a new .conf file from the template.
 
-	`sudo vim /etc/apache2/sites-available/001-devcycle.conf`.
-7. Copy and paste the following, you may edit these values if desired (such as where to collect static files).
+ ```
+sudo cp /usr/local/devcycle/001-devcycle.conf.template /etc/apache2/sites-available/001-devcycle.conf
+sudo vim /etc/apache2/sites-available/001-devcycle.conf
+ ```
+ * Change "SERVER_NAME" to match your server's name ex: `devcycle`
+ * Change "SERVER_ADDRESS" to match your domain ex: `devcycle.se.rit.edu`
+ * Change "SERVER_EMAIL_ADDRESS" to match your email address ex: `pandaman@example.com`
 
-```
-WSGIPythonPath usr/local/devcycle
-
-<VirtualHost *:80>
-        ServerName devcycle.se.rit.edu
-        ServerAlias devcycle.se.rit.edu
-        ServerAdmin someaddress@example.com
-
-        Alias /static /var/www/static/
-
-        <Directory /public/static/>
-                Order allow,deny
-                Allow from all
-        </Directory>
-
-        WSGIDaemonProcess devcycle processes=2 threads=15 display-name=%{GROUP} python-path=/usr/local/devcycle/
-        WSGIProcessGroup devcycle
-        WSGIScriptAlias / /usr/local/devcycle/dataCollection/wsgi.py
-
-        <Directory /usr/local/devcycle/dataCollection>
-                <Files wsgi.py>
-                Order deny,allow
-                Allow from all
-                </Files>
-        </Directory>
-</VirtualHost>
-```
+7. Create and Change the application settings:
 
-* Create and Change the application settings:
-
-```
+ ```
 sudo cp /usr/local/devcycle/dataCollection/settings.py.template /usr/local/devcycle/dataCollection/settings.py
 sudo vim /usr/local/devcycle/dataCollection/settings.py
-```
+ ```
+ * Under DATABASES, modify USER and PASSWORD fields to reflect the database user you created in "Install The Database" (above).
```

Comment:
Nitpicky; perhaps add a link back to relevant section [as such](https://github.com/cman131/devcycle-server/blob/staging/README.md#install-postgres-with-postgis-install-the-database)

Or maybe even indicate that it was in step 4.